### ISSUE TITLE
Customize tweet data

### ISSUE THREAD

**Issue by CorentinMic** [janezkranjc/twitter-tap#6]
Is there a way to configure tweets attributes to be stored with tap ? Or do we have to use Tweetset for that ?

**Comment by janezkranjc** [comment]
by default everything that is collected from twitter is stored in mongodb, my guess is you would like to filter data?

**Comment by CorentinMic** [comment]
Exactly, I would like not to collect every info about a tweet but choose only few ones.

**Comment by CorentinMic** [comment]
Is that possible ?
